internal/api/v1/services/instance.go

```diff
@@ -13,13 +13,21 @@ import (
 	"github.com/celestiaorg/talis/internal/types/infrastructure"
 )
 
-// JobServiceInterface defines the interface for job operations.
-// This interface is used by InstanceService to manage job states during infrastructure operations.
-// It provides a minimal set of methods needed to create and update job statuses,
-// following the Interface Segregation Principle by only exposing the methods that are actually used.
+// InstanceServiceInterface defines the interface for instance operations
+type InstanceServiceInterface interface {
```

Comment:
same here, it is an interface by definition. we don't need interface in the name